.github/workflows/generate.yml

```diff
@@ -1,18 +1,28 @@
-name: Generate configs and trigger network restart
+name: Generate config
```

Comment:
Please change name to proper one: `Generate configs and trigger network restart`